rpc/handlers_test.go

```diff
@@ -1846,7 +1846,7 @@ func TestTransactionStatus(t *testing.T) {
 		{
 			network:           utils.MAINNET,
 			verifiedTxHash:    utils.HexToFelt(t, "0xf1d99fb97509e0dfc425ddc2a8c5398b74231658ca58b6f8da92f39cb739e"),
-			nonVerifiedTxHash: utils.HexToFelt(t, "0x6c40890743aa220b10e5ee68cef694c5c23cc2defd0dbdf5546e687f9982ab1"),
+			nonVerifiedTxHash: utils.HexToFelt(t, "0x6cd1526afa9e294bfccf58e0fc73f0486175bbfed11d6bf661c129d3cc8b9f0"),
```

Comment:
Can you please provide some context for this change?

Reply:
Oops, I was just testing something locally, and forgot to reset it back.